### ISSUE TITLE
Fix Flaky Behavior in `getVolunteerRanks` Query Implementation

### ISSUE THREAD

**Issue by prayanshchh** [PalisadoesFoundation/talawa-api#2781]
**Describe the bug**
``
**To Reproduce**

1. run npx vitest tests/resolvers/Query/getVolunteerRanks.spec.ts

**Expected behavior**
all tests should pass

**Actual behavior**
` getVolunteerRanks for monthly, descending, name, no limit`  test is failing

**Screenshots**
![image](https://github.com/user-attachments/assets/42b2354a-32d2-4274-8a18-6f5d15bd849b)
**Additional details**

the issue is in the implementation of `getVolunteerRanks` query rather than the test itself
**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by prayanshchh** [comment]
assign

**Comment by Aad1tya27** [comment]
The issue has already been fixed in the PR #2770

**Comment by Aad1tya27** [comment]
I think the actual answer should be 6 only, I've explained it in a comment in my pr. I had changed the expected answer to be equal to 6 in it.

**Comment by prayanshchh** [comment]
> I think the actual answer should be 6 only, I've explained it in a comment in my pr. I had changed the expected answer to be equal to 6 in it.

yes u are right, the implemetation seems fine, I slacked @varshith257
